backend/cypress/support/dissemination-table.js

```diff
@@ -34,3 +80,51 @@ export function testReportIdFound(reportId) {
     expect(hasAgency).to.be.true;
   });
 }
+
+export function testReportIdFoundWithTribalAccess(reportId) {
+  const tribal_access_email = `${crypto.randomUUID()}@example.com`;
+  const tribal_access_user_id = crypto.randomUUID();
+
+  grantTribalAccess(tribal_access_email, tribal_access_user_id);
+  
+  // try to pull the tribal, non-public data from the API using the (now) privileged user
+  cy.request({
+    method: 'GET',
+    url: `${API_GOV_URL}/general`,
+    headers: {
+      Authorization: `Bearer ${API_GOV_JWT}`,
+      'X-Api-Key': API_GOV_KEY,
+      'X-Api-User-Id': tribal_access_user_id,
+    },
+    qs: {report_id: `eq.${reportId}`},
+  }).should((response) => {
+    expect(response.body).to.have.length(1);
+    const hasAgency = !!(response.body[0]?.cognizant_agency || response.body[0]?.oversight_agency);
```

Comment:
It would be nice if this line was obviously clear.

https://stackoverflow.com/questions/29312123/how-does-the-double-exclamation-work-in-javascript

It isn't to me.

Reply:
And, should we be confirming the value of `is_public` while doing this?

Reply:
Would it be better to just add comments on what the code is doing?